client/src/index.tsx

```diff
@@ -25,10 +27,17 @@ const GlobalStyle = createGlobalStyle`
   }
 `;
 
-const httpLink = createHttpLink({
+const httpLink = new HttpLink({
   uri: `${config.serverUrl}/graphql`,
 });
 
+const wsLink = new WebSocketLink({
+  uri: `ws://localhost:4000/graphql`,
```

Comment:
if we harcode the value it won't work when deployed to an external server (on prod)

Reply:
Maybe we can use `config.serverUrl` but we need to replace the HTTP with ws

Reply:
Actually, this won't work `client/src/config.ts`, we need to find a good way to make in work on prod

Reply:
Resolving this here as new issue is already opened.